DatabaseV2.SQL

jwwyeth opened the pull request:
Updated database schema please review and offer feedback if needed.

jwwyeth committed:
DatabaseV2.SQL

Heckart commented on database.sql:
```diff
@@ -1,23 +1,100 @@
+    OrderID TEXT UNIQUE NOT NULL,
+    UserID TEXT NOT NULL,
+    BusinessID TEXT NOT NULL,
+    ShopperID TEXT NOT NULL,
```
`ShopperID` needs to be `NULL`able because an entry to this table will be made before a shopper is selected.

Heckart commented on database.sql:
```diff
@@ -1,23 +1,100 @@
+    BusinessID TEXT NOT NULL,
+    ShopperID TEXT NOT NULL,
+    Status TEXT,
+    fulfillmentTime INT NOT NULL,
```
`fulfillmentTime` also needs to be `NULL`able because a fulfillment time is not created until a shopper is selected.

Heckart commented on database.sql:
```diff
@@ -1,23 +1,100 @@
+
+CREATE TABLE IF NOT EXISTS JobBoard(
+    ShopperID TEXT UNIQUE NOT NULL,
+    fulfillmenttime TEXT UNIQUE NOT NULL,
```
Do you mind going through and making consistent variable capitalization? The backend currently assumes camelCase for all names

ex: "addressPrimary," "fulfillmentTime," "state," "city"

jwwyeth committed:
Update database.sql

I have camelcased all values and made fullfimentime and shopperid nullable in orders table

Heckart commented:
thanks for the fixes, approved.
